### ISSUE TITLE
Package with '-' (dash) in name create '_' underscore in wheel file name. (DOC ISSUE)

### ISSUE THREAD

**Issue by matthewsht** [pypa/build#439]
I have a namespaced package called "orgname-thing". 
Setup.cfg has name as orgname-thing

Everything works as expected except:
on build, I get a file name "orgname-thing-version.tar.gz" -- expected that.
"orgname_thing-version-py3-none-any.whl" --- did not expect that.

I'm fairly certain this is the INTENDED behavior - my "issue" is I can't figure out why. I've consulted builds docs, googled for PEP's and have no clue.

(In addition, pip with a extra_index_url looks for "orgname-thing/orgname_thing*whl" which was REALLY surprising.)

In short, I'm asking for docs on why this is intended.

**Comment by FFY00** [comment]
This is specified by the wheel spec.

https://www.python.org/dev/peps/pep-0427/#escaping-and-unicode

**Comment by matthewsht** [comment]
Still hope for an addition to the official docs, but perhaps this bug itself will be sufficient. 
Thank You for the library.

**Comment by layday** [comment]
FWIW sdists should follow the same rule to separate the name from the version but setuptools doesn't for legacy reasons :)
